### DISCUSSION TITLE
[A-L] (2023/24) Foglio 4 - Esercizio 15

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#303] [Esercizi - Viaggi/Piazza] [answered]
![Screenshot 2023-11-02 185217](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/d90e93af-0b30-4404-80a7-47743a4c84fa)
---
![Screenshot 2023-11-02 185208](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/8e606fe6-75ca-4d22-a8eb-7c8153202fc9)

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/65602628-cbe4-45fd-b3cc-07fba8e81028)

**Comment by daromacarlo** [comment, marked as answer]
punto 1 analogo a @CiottoloMaggico . 
punto 2 differente.
![Immagine 2023-11-06 182217](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/58a18c1f-36a9-43f3-825d-f981ab84af56)
